src/test/java/com/artipie/rpm/TestRpm.java

```diff
@@ -116,4 +113,53 @@ public final void put(final Storage storage) throws IOException {
             ).join();
         }
     }
+
+    /**
+     * An invalid rpm.
+     * @since 0.9
+     */
+    final class Invalid implements TestRpm {
+        @Override
+        public void put(final Storage storage) throws IOException {
+            storage.save(
+                new Key.From("invalid.rpm"),
+                new Content.From(new byte[] {0x00, 0x01, 0x02 })
+            ).join();
+        }
+    }
+
+    /**
+     * Multiple test rpms.
+     * @since 0.9
+     */
+    final class Multiple implements TestRpm {
+
+        /**
+         * Rpms.
+         */
+        private final Iterable<TestRpm> rpms;
+
+        /**
+         * Ctor.
+         * @param rpms Rpms.
+         */
+        public Multiple(final TestRpm... rpms) {
```

Comment:
@paulodamaso @victornoel the idea of this interface and it's implementations is to avoid code duplications on adding these test packages to storage in different tests. For example, to add one package to storage we will constantly repeat
```
storage.save(
     new Key.From(key, RpmTest.ABC.getFileName().toString()),
     new Content.From(Files.readAllBytes(RpmTest.ABC))
).join() 
```
along with the path to the rpm file. Also it's easy to forget to call `join()` or `add()` on `Completable` which leads to possible test failure. So, I've decided to introduce these classes to avoid repeating this code and possible mistakes.
For now we have only 2 rpms in test resources, later if we need more we simply add another implementation and so on.   
Does it make sense?

Reply:
@olenagerasimova it does make sense to me, I like it :)